src/SQLQueryStress/Form1.cs

```diff
@@ -472,13 +472,25 @@ private void UpdateUi()
 
         private void btnCleanBuffer_Click(object sender, EventArgs e)
         {
+            if (!_settings.MainDbConnectionInfo.TestConnection())

```

Comment:
Did you forget to rebase on the latest master?

Reply:
Sorry, I didn't realise I needed to do a rebase, learning as I go. Rebased my fix-warnings branch on my master now.